specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/storage.json

```diff
@@ -3290,7 +3296,7 @@
           "items": {
             "type": "string"
           },
-          "description": "An array of predefined enum values. Only blockBlob is supported."
+          "description": "An array of predefined enum values. Currently blockBlob supports all tiering and delete actions. Only delete actions are supported for appendBlob."
```

Comment:
Thanks for the update!
Would you please also add the sample usage to input append blob and block blob together?
This can't get directly from the description of the item.

Reply:
added one example